views/js/qtiCommonRenderer/helpers/Graphic.js

```diff
@@ -393,33 +382,34 @@ define([
          * @param {Function} [options.create] - call once created
          * @param {Function} [options.remove] - call once removed
          */
-        createTarget : function createTarget(paper, options){
-            var self    = this;
-            options     = options || {};
-            var point   = options.point || {x : 0, y : 0};
-            var baseSize= 18;
-            var factor  = (paper.w && paper.width) ? paper.width / paper.w : 1;
-            var size    = factor !== 1 ? Math.floor(18 / factor) + 1 : baseSize;
-            var half    = size / 2;
-            var x       = point.x >= half ? point.x - half : 0;
-            var y       = point.y >= half ? point.y - half : 0;
-            var hover   = typeof options.hover === 'undefined' ? true : !!options.hover;
+        createTarget : function createTarget(paper, options) {
+            options      = options || {};
```

Comment:
We are used to declare variables to reflect actual hoisting, and the rule is to fix them when you modify a piece of code :
 
```js
createTarget : function createTarget(paper, options) {
    var self = this;
    var factor = paper.width / paper.w;
    var point;
    var hover;
    ...
   options      = options || {};
   point          = options.point ||  {x : 0, y : 0};
```